### PR TITLE
HCX Integration - Claims Disabled

### PR DESCRIPTION
**Pull request by khavinshankar**
- [ ] **Change the proxy before merging**

### CODE REVIEW

**Comment by nihal467** [comment]
@khavinshankar 
image

- [ ] in the patient creation page, it is asking for an insurance detail 

image

- [ ] in the patient consultation page, I am able to see the claims buttons

image

- [ ] in the patient discharge pop-up, I can see the claim insurance page

File: .env [hunk 1]
```diff
@@ -6,4 +6,7 @@ REACT_APP_COVER_IMAGE=https://cdn.coronasafe.network/care_logo.svg
 REACT_APP_COVER_IMAGE_ALT=https://cdn.coronasafe.network/care_logo.svg
 
 # Dev envs
-ESLINT_NO_DEV_ERRORS=true
\ No newline at end of file
+ESLINT_NO_DEV_ERRORS=true
+
+# App envs
+REACT_APP_ENABLE_HCX=true
```
Comment (khavinshankar) [inline .env]:
```suggestion
REACT_APP_ENABLE_HCX=false
```

**Comment by nihal467** [comment]
@khavinshankar @gigincg @aparnacoronasafe 
image

when we create a new patient with insurance detail-filled, there is a notification saying checking policy eligibility in the consultation creation form, Is it intentional behavior?

Other than that, this PR LGTM,

**Comment by mathew-alex** [comment]
@khavinshankar - delete the below files and update the PR.

- .github/workflows/deply-hcx.yaml 
- aws/frontend_hcx.json

File: src/Components/Common/prescription-builder/ProcedureBuilder.tsx [hunk 2]
```diff
@@ -106,10 +106,12 @@ export default function ProcedureBuilder(props: Props<ProcedureType>) {
                       }}
                     />
                   </div>
+                  ;
```
Comment (rithviknishad) [inline src/Components/Common/prescription-builder/ProcedureBuilder.tsx]: image

```suggestion
```

**Pull request merged**